python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -707,7 +730,7 @@ def readSrcMatches(self, dataRefList, dataset, hscRun=None, wcs=None, aliasDictL
         if not catList:
             raise TaskError("No matches read: %s" % ([dataRef.dataId for dataRef in dataRefList]))
 
-        return concatenateCatalogs(catList)
+        return concatenateCatalogs(catList), areaDict
```

Comment:
Is areaDict now made twice? Once in readCatalogs and once in readSrcMatches?

Reply:
If you think it is necessary to make it twice then that's fine. Is there anyway to only do it once?